test/e2e/test_system.py

```diff
@@ -542,3 +545,38 @@ def test_delete_system_with_child_system(test_client):
 
     assert response.status_code == 409
     assert response.json()["detail"] == "System has child elements and cannot be deleted"
+
+
+def test_delete_system_with_child_item(test_client):
```

Comment:
Can we have another test with child system?

Reply:
That one is above (it already existed)

Reply:
Ah my bad, sorry.